### PR TITLE
Move repository config to a json file for latest read.

### PR DESCRIPTION
**Pull request by abhigyank**
Move the data of repositories to  a json file that is read on every request so that we don't have to restart server every time we add a new repository.

### CODE REVIEW

**Comment by abhigyank** [comment]
@Milind712000  I have not tested this, it'd be good if you can test this for any errors before merging.

File: conf/repositoryList.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+		'name':'test repository',
+		'path':'/home/milind/Documents/projects/test-repo',
+		'secret':'lvgshkvh750945'
+	},
+	'2':{
+		'name':'name of 2nd repository',
+		'path':'/df/adfs/fda/dfa/',
+		'secret':'as435djfav'
+	}
+}
```
Comment (Milind712000) [inline conf/repositoryList.json]: Valid JSON requires double quotes

File: conf/repositoryList.js [hunk 2]
```diff
@@ -1,12 +1,8 @@
...
-		'name':'name of 2nd repository',
-		'path':'/df/adfs/fda/dfa/',
-		'secret':'as435djfav'
-	}
-}
+const fs = require('fs');
+
+
+module.exports = function() {
+	let rawdata = fs.readFileSync('./repositoryList.json');
```
Comment (Milind712000) [inline conf/repositoryList.js]: the path will be ./conf/repositoryList.json
as the starting directory for fs module will be the directory node was started from (the apps root directory)

**Comment by abhigyank** [comment]
@Milind712000 Updated. Can you do a review now?

**Pull request merged**